libs/ui/src/components/atoms/Checkbox/Checkbox.styles.ts

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components';
+
+import { font } from '../../../theme/global/font';
+import { Theme } from '../../../types/theming';
+
+import * as CheckboxPrimitive from '@radix-ui/react-checkbox';
+
+export const StyledCheckbox = styled(CheckboxPrimitive.Root)`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: transparent;
+  border: 2px solid
+    ${(props) =>
+      props.checked
+        ? props.theme.checkbox.activeBorder
+        : props.theme.checkbox.border};
+  border-radius: 1px;
+  width: 18px;
+  height: 18px;
```

Comment:
Shouldn't we be using rems over px's? @jordanlesich Is that correct?

Reply:
Yep!

Reply:
Done